presto-main/src/test/java/io/prestosql/sql/planner/optimizations/TestRemoveUnsupportedDynamicFilters.java

```diff
@@ -116,7 +116,8 @@ public void testUnconsumedDynamicFilterInJoin()
                 join(
                         INNER,
                         ImmutableList.of(equiJoinClause("ORDERS_OK", "LINEITEM_OK")),
-                        filter("ORDERS_OK > 0",
```

Comment:
i prefer the previous formatting

i know it does not conform to our global formatting rules, but i find this way more readable in plan tests